L1Trigger/TrackerTFP/src/DataFormats.cc

```diff
@@ -69,6 +69,11 @@ namespace trackerTFP {
     numStreamsTracks_ = vector<int>(+Process::end, 0);
     numStreamsTracks_[+Process::kfin] = numStreams_[+Process::kf];
     numStreamsTracks_[+Process::kf] = numStreams_[+Process::kf];
+    // Print digi data format of all variables of any specified algo step
```

Comment:
Please remove the commented portions.

Reply:
The few lines of commented out code, illustrate how to print out the data format configuration of any chosen processing step. They are therefore useful.